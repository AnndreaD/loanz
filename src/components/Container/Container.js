import React from 'react';
import './Container.scss';
import PropTypes from 'prop-types';

const Container = ({ children, title }) => {
    return (
        <div className="container">
            <h1 className="container-title">{title}</h1>
            {children}
        </div>
    );
};

export default Container;

Container.propTypes = {
    title: PropTypes.string,
};
