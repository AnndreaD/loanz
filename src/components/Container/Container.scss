@import '../../colors';

.container {
    height: 40%;
    width: 70%;
    background: $background;
    border-radius: 2px;
    margin-left: 15%;
    margin-top: 10%;
    padding: 2rem;

    .container-title {
        color: $green-alternate;
    }
}
