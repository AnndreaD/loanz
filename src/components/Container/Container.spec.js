import Container from './Container';
import renderer from 'react-test-renderer';
import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import expect from 'expect';
import '@testing-library/jest-dom';

configure({ adapter: new Adapter() });

describe('Container', () => {
    it('SNAPSHOT: should render expected content', () => {
        const defaultTree = renderer.create(<Container />);
        expect(defaultTree.toJSON()).toMatchSnapshot();
    });
});
