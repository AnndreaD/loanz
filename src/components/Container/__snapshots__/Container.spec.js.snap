// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container SNAPSHOT: should render expected content 1`] = `
<div
  className="container"
>
  <h1
    className="container-title"
  />
</div>
`;
